external/@wordpress/blocks/index.js

```diff
@@ -19,14 +28,20 @@ export function registerBlock( namespace, block, settings ) {
  * @return {?Object}           Block settings
  */
 export function getBlockSettings( namespace, block ) {
+	if ( ! blocks[ namespace ] || ! blocks[ namespace ][ block ] ) {
+		return null;
+	}
 
+	return blocks[ namespace ][ block ];
 }
 
 /**
  * Returns all registered blocks.
  *
- * @return {Object} Block settings keyed by block name
+ * @return {Array} Blocks settings
```

Comment:
Repeating my comment elsewhere, I think we can let go of namespace. The name should be unique and descriptive anyways because the user would interact and search for it.